### PR TITLE
fix typos in qa-scripts

### PR DESCRIPTION
**Pull request by philbucher**
#### Description of the feature or the bug
Typos and copy-paste error in qa scripts can lead to confusion

#### Description of the changes
Fixed small typos and copy-paste error in qa-scripts

### CODE REVIEW

**Review by sebastienVilleneuve — APPROVED** [review]
Hi @philbucher 

thanks so much for your contribution.

Regards,
Sébastien

File: qa-tests/test_script.bat [hunk 1]
```diff
@@ -80,7 +80,7 @@ GOTO ARG_LOOP
 
 
 
-REM As this is a bash script it is intended to be executed on Linux Plarforms.
+REM As this is a bash script it is intended to be executed on Windows Platforms.
```
Comment (sebastienVilleneuve) [inline qa-tests/test_script.bat:83]: Hi , I think initial comment was correct here

Sébastien

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]

File: qa-tests/test_script.bat (hunk 1 above)
Comment (philbucher) [reply qa-tests/test_script.bat:83]: Hi,
in the next line it says `test_directory=ctest_suite_win64`
also to my knowledge `bat` scripts are used only on win

But I am not a win expert, I can revert

Comment (sebastienVilleneuve) [reply qa-tests/test_script.bat:83]: Hi , ok , I missed the name of the script , in this case this should be '**bat** script it is intended to be executed on **Windows Platforms**'

Thanks,
Sébastien

**Review by sebastienVilleneuve — APPROVED** [review]

**Comment by sebastienVilleneuve** [comment]
@philbucher 

last remark : we only accept PR with 1 commit , can you quash your commits ?

Sébastien

**Comment by philbucher** [comment]
it is one commit now, but not sure if force-pushing is the best solution

**Comment by sebastienVilleneuve** [comment]
you should use : 
git rebase -i main

**Comment by sebastienVilleneuve** [comment]
It is ok now, Thanks

**Comment by sebastienVilleneuve** [comment]
It is ok now, Thanks

**Pull request merged**